Content.Server/Administration/AdminVerbSystem.cs

```diff
@@ -0,0 +1,199 @@
+using Content.Server.Administration.Commands;
+using Content.Server.Administration.Managers;
+using Content.Server.Administration.UI;
+using Content.Server.Configurable;
+using Content.Server.Disposal.Tube.Components;
+using Content.Server.EUI;
+using Content.Server.Ghost.Roles;
+using Content.Server.Inventory.Components;
+using Content.Server.Mind.Commands;
+using Content.Server.Mind.Components;
+using Content.Server.Players;
+using Content.Server.Verbs;
+using Content.Shared.Administration;
+using Content.Shared.Chemistry.Components.SolutionManager;
+using Content.Shared.Interaction.Helpers;
+using Content.Shared.Notification.Managers;
+using Content.Shared.Verbs;
+using Robust.Server.Console;
+using Robust.Server.GameObjects;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Localization;
+
+namespace Content.Server.Administration
+{
+    /// <summary>
+    ///     System to provide various global admin/debug verbs
+    /// </summary>
+    public class AdminVerbSystem : EntitySystem
```

Comment:
For now I've just placed most of the various admin debug-verbs into this system, despite the fact that some of them could also just go into their own systems (e.g., control mob and make sentient could go into the mind system).

I'm not sure which is preferred. Either way, this system would still have to exist for some verbs that have no other clearly associated system (e.g., "delete" & "Rejuvenate").